RomanASP.py

```diff
@@ -101,6 +101,11 @@ def main():
                         help='end of desired light curve in days from peak.',
                         default=np.inf)
 
+    parser.add_argument('--object_type', type=str, required=False,
+                        help='If star, will run on stars. If SN, will run  ' +
+                             'on supernovae. If None, assumes supernova.',
```

Comment:
I would suggest forcing this to be either "star" or "SN".  Don't allow it to be anything else, including None.  

`parser.add_argument(..., choices=["star", "SN"], default="SN", help=...)`